packages/editor/src/components/word-count/index.js

```diff
@@ -2,11 +2,12 @@
  * WordPress dependencies
  */
 import { withSelect } from '@wordpress/data';
+import { _x } from '@wordpress/i18n';
 import { count as wordCount } from '@wordpress/wordcount';
 
 function WordCount( { content } ) {
 	return (
-		<span className="word-count">{ wordCount( content, 'words' ) }</span>
+		<span className="word-count">{ wordCount( content, _x( 'words', 'Word count type. Do not translate!' ) ) }</span>
```

Comment:
For reference, this is the description from core:

```php
/*
 * translators: If your word count is based on single characters (e.g. East Asian characters),
 * enter 'characters_excluding_spaces' or 'characters_including_spaces'. Otherwise, enter 'words'.
 * Do not translate into your own language.
 */
```

Reply:
Ah, you were just faster. :)